docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #90.        return nn_distance, bsf
</pre><p>We may combine the two <code>if / break</code> blocks  and we can just return <code>nn_distance</code> which is either exact [if it is greater than bfs] or approx [if otherwise]. We may even return the <code>is_exact</code> flag to make it more explicit. Then, we can update <code>bfs</code> outside of this function. </p><p> </p><p>[I am trying to see if I can find a way to return one object rather than two objects] </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Returning 2 items is fine I think?</p>

Reply:
Going to leave it as is. [Also: Now that I think more about it, it is better to remain faithful to the algorithm as much as possible in the first stage of our work. And then, if needed, we can revise it to improve it]